.fastly/config.toml

```diff
@@ -8,6 +8,10 @@ remote_config = "https://developer.fastly.com/api/internal/cli-config"
 ttl = "5m"
 
 [language]
+  [language.go]
+    tinygo_constraint = ">= 0.24.0"
+    toolchain_constraint = ">= 1.17"
```

Comment:
Do we want to also include an upper limit here?

Reply:
We don't for other languages so I'd be hesitant to introduce one here unless we feel there is likely to be a breaking change result from using (for example go 1.19)

Reply:
YMMV but I think we should play it safe for the first release.